src/View/HelperTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
Doc block with copyright and licence is missing. An example can be found here:

https://github.com/zendframework/maintainers/blob/abab4a99f3e6d1fac4d7400232ce30ee0bc95249/template/src/ConfigProvider.php#L2-L6